src/pyscipopt/scip.pyx

```diff
@@ -3091,7 +3170,7 @@ cdef class Model:
         cdef SCIP_BENDERSCUT* scip_benderscut
         scip_benderscut = SCIPfindBenderscut(_benders, n)
         benderscut.model = <Model>weakref.proxy(self)
-        benderscut.benders = <Benders>weakref.proxy(benders)
+        benderscut.benders = benders
```

Comment:
why this change? was `benders` getting freed before it should?

Reply:
I can't remember. I would have to retest

Reply:
I think that we use weakrefs in model because otherwise we create a circular dependency and nothing gets freed (not sure though). Does this not happen here?

Reply:
I just tested this and the error that i get is
```
TypeError: Argument 'benders' has incorrect type (expected pyscipopt.scip.Benders, got weakproxy)
```
It seems that it doesn't work to use the weakref in this settiing.

Reply:
that sounds weird. how can I reproduce that?